scripts/optimize-svg.js

```diff
@@ -0,0 +1,108 @@
+import fs from 'fs';
+import path from 'path';
+import { optimize } from 'svgo';
+
+const directories = ['./chains', './deployments'];
+const MAX_FILE_SIZE = 100 * 1024; // 100KBs
+const RASTER_IMAGE_REGEX = /<image[^>]*>/i;
+
+function isValidSvg(filePath) {
+  const fileName = path.basename(filePath);
+  const stats = fs.statSync(filePath);
+  const currentFileSize = (stats.size / 1024).toFixed(2);
+
+  if (!fileName.endsWith('logo.svg')) {
+    console.error(`Error: File does not end with 'logo.svg' -> ${filePath}`);
+    process.exit(1);
+  }
+
+  if (stats.size > MAX_FILE_SIZE) {
+    console.error(
+      `Error: File size exceeds 100KBs (Current size: ${currentFileSize}KB) -> ${filePath}`,
+    );
+    process.exit(1);
+  }
+
+  const fileContent = fs.readFileSync(filePath, 'utf8');
+  if (RASTER_IMAGE_REGEX.test(fileContent)) {
+    console.error(
+      `Error: File contains an <image> tag, likely embedding a raster image -> ${filePath}`,
+    );
+    process.exit(1);
+  }
+}
```

Comment:
I think it would be better if we catch all the invalid SVGs and print one output at the end, instead of failing fast - wdyt?

Reply:
Done, instead of exiting, it just prints the error and then exit at the end if there an error